stack/lab/Dockerfile

```diff
@@ -105,3 +105,9 @@ ENV NOTEBOOK_ARGS \
      "--MappingKernelManager.cull_interval=300" \
      "--TerminalManager.cull_inactive_timeout=600" \
      "--TerminalManager.cull_interval=60"
+
+# Set up the logo of notebook interface
+COPY --chown=${NB_UID}:${NB_GID} aiidalab-wide-logo.png ${CONDA_DIR}/notebook-logo.png
```

Comment:
Why do you put the logo in `$CONDA_DIR`? Isn't `/opt/` a better place? Perhaps `/opt/static/`?

Reply:
Could be, I'll use `/opt`, it is a temporary location, and I think it is better to use `mv` instead of `cp`.

Reply:
Can't use `/opt` since I didn't run `fix-permission`. I use `/tmp` instead, so I can use `mv` to avoid